src/sdk/model/vdc/distributed-firewall/__json__/distributed-firewall-layer2-json.ts

```diff
@@ -0,0 +1,7 @@
+import { DistributedFirewallJson } from './distributed-firewall-json';
+
+/**
+ * Distributed Firewall Layer 2 Object JSON (Ethernet).
+ */
+export interface DistributedFirewallLayer2Json extends DistributedFirewallJson {
```

Comment:
Extending this interface with what? Or for what purpose?

Reply:
The idea behind this is:
- We don't know how NSX API and objects will evolve in future, DistributedFirewall Layer2 and Layer3 are different objects with separated endpoints, for the time being objects are using the same parameters (inherited from DistributedFirewallJson) but we don't know if it will remains like that, so if one or more params are added to Layer2 or Layer3 we can add them in these classes.

Reply:
Ah gotcha. Thanks for the explanation @fradinni